common/utils/allowCors.ts

```diff
@@ -0,0 +1,22 @@
+import { NextApiRequest, NextApiResponse } from 'next';
+export const allowCors =
```

Comment:
are we using this?

Reply:
I had this created just in case, but it seems like it may not be necessary anymore.